spec/factories/users.rb

```diff
@@ -31,6 +33,8 @@
 #
 FactoryBot.define do
   factory :user do
+    first_name { Faker::Internet.name }
+    last_name { Faker::Internet.name }
```

Comment:
So you can check what `Faker` is doing in the console.
```rb
# open a rails console
require 'faker' # be sure to include this line
Faker::Internet.name
#-> "Faker::Internet"
```
This just returns the name of the class, which is standard Ruby behavior
```rb
Array.name
# -> "Array"
```
But we want it to be coming up with a name. Faker has a name module that handles this nicely
```rb
Faker::Name.first_name
# -> "Shirly"
Faker::Name.last_name
# -> "Towne"
```

Reply:
This is what I intended on discussing during our call today. I will take a look at these issues.